src/actions/AmountActions.js

```diff
@@ -3,6 +3,15 @@ import BN from 'bignumber.js';
 import { setFormValue, setFormError, setValue } from './FormActions';
 import { ECHO_ASSET_ID } from '../constants/GlobalConstants';
 
+/**
+ * @method amountInput
+ *
+ * @param {String} form
+ * @param {String} value
+ * @param {Object} currency
+ * @param {string} name
+ * @returns {undefined}
```

Comment:
its return function, let it will be like @returns `{function(dispatch, getState): Promise<undefined>}`
also change to this return type in other actions